functions/task3.js

```diff
@@ -0,0 +1,11 @@
+const recursiveFunction = (num) => {
```

Comment:
`recursiveFunction` name does not represent what function actually does. Use more informative name like `countDown`